### PR TITLE
Fixes the issues highlighted in #27 and fixes the issues within PR #30

### PR DESCRIPTION
**Pull request by jwhitt3r**
This PR addresses the issues outline in Fix readme https://github.com/trypear/pearai-app/issues/27

I have reworked the page to link to the master document, included the contributing section to align with the contents of the document.

Updated the features to be inline with the webpage
Added a roadmap linking the master document
Added a license section linking to your license model, I have removed the MIT license at the top as that is confusing with the actual document
Added a contact section
Added an acknowledgement section listing contributors
Reworded the opening paragraph
Reworked the page to include the Prerequisites of an OpenAI API key

This relinks the work in PR #30

### CODE REVIEW

**Comment by jwhitt3r** [comment]
Not entirely sure what I need to do to fix this issue, any help would be appreciated.

File: README.md [hunk 1]
```diff
@@ -1,77 +1,72 @@
...
+  path: '/code/pearai/out/vs/code/electron-main/main.js',
   phase: 'loading',
   moduleId: 'vs/code/electron-main/main',
   neededBy: [ '===anonymous1===' ]
 }
 ```
-- Remove the build and re-ran script `rm -rf out`
-- `./scripts/code.sh`
+To resolve this, follow the below steps:
+ 1. Remove the build and re-ran script `rm -rf out`
```
Comment (Fryingpannn) [inline README.md:52]:
```suggestion
 1. Remove the build `rm -rf out`
```

**Comment by Fryingpannn** [comment]
We're looking into it. You had reset your repo after https://discord.com/channels/1237009981803462729/1237018645884960868/1242685139570528287 correct?

**Review by Fryingpannn — COMMENTED** [review]
It may be because of the # sign in your branch name. Github could have trouble finding your remote branch because of the special character. Let's try renaming your branch:

image

```
# Rename the branch locally
git checkout fix-issues-#27-and-#30
git branch -m fix-issues-27-and-30

# Push the renamed branch and delete the old branch on remote
git push origin --delete fix-issues-#27-and-#30
git push origin fix-issues-27-and-30

# Set upstream for the new branch
git push --set-upstream origin fix-issues-27-and-30
```

File: README.md (hunk 1 above)
Comment (jwhitt3r) [reply README.md:52]: Resolved

**Comment by jwhitt3r** [comment]
I followed the steps above and I am still getting the same error

**Pull request closed without merging**